app/js/lib/reducers/form_content.js

```diff
@@ -25,6 +35,32 @@ function stateContainsUnknownKeys( state ) {
 	return !_.isEmpty( getInvalidKeys( state ) );
 }
 
+function clearFieldsIfAddressTypeChanges( newState, payload ) {
+	if ( payload.contentName !== 'addressType'  ) {
+		return;
+	}
+	switch ( payload.value ) {
+		case 'privat':
```

Comment:
Few seconds later: wait, I have just noticed above that in ` app/js/lib/reducers/form_content.js` there is line 11:
```js
addressType: 'person', // person, firma and anonym
```
I'm confused. Shouldn't this be `person` instead of `privat` then?